app/lib/features/chat_ng/widgets/events/image_message_event.dart

```diff
@@ -0,0 +1,170 @@
+import 'dart:io';
+
+import 'package:acter/common/models/types.dart';
+import 'package:acter/common/toolkit/errors/inline_error_button.dart';
+import 'package:acter/common/widgets/image_dialog.dart';
+import 'package:acter/features/chat/models/media_chat_state/media_chat_state.dart';
+import 'package:acter/features/chat/providers/chat_providers.dart';
+import 'package:acter_flutter_sdk/acter_flutter_sdk_ffi.dart' show MsgContent;
+import 'package:flutter/material.dart';
+import 'package:flutter_chat_ui/flutter_chat_ui.dart';
+import 'package:flutter_gen/gen_l10n/l10n.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'package:phosphor_flutter/phosphor_flutter.dart';
+
+class ImageMessageEvent extends ConsumerWidget {
```

Comment:
The inner implementation is same except that we now access ```MsgContent``` and identifiers directly instead of old ```types.Message```